### ISSUE TITLE
Display a link to /terms and /privacy in the first screen of the onboarding tour

### ISSUE THREAD

**Issue by ckprice** [mozilla-services/screenshots#2499]
This wants to be added to the first screen of the onboarding tour (Welcome.png) as shown in https://github.com/mozilla-services/screenshots/issues/2307#issuecomment-286371841.

Text from legal is:

> By using Firefox Screenshots, you agree to the Screenshots [Terms](#) and [Privacy Notice](#)

**Comment by wresuolc** [comment]
This should go on the bottom of the Welcome slide, centered, same font, maybe -1 size.

https://www.dropbox.com/sh/zkb6xrkkv9wusoz/AACPnP31HAn7RIoKrDYd8UnDa/1.1%20Welcome.png?dl=0

Worst case scenario:
By using Firefox Screenshots, you agree to the <a href="%LINK1%">Screenshots Terms</a> and <a href="%LINK2%">Privacy Notice</a>

**Comment by ianb** [comment]
From flod: `By using Firefox Screenshots, you agree to the Screenshots {link_terms} and {link_privacy}` + strings for Terms and Privacy Notice (but shouldn't reuse those strings from elsewhere)

**Comment by 6a68** [comment]
@ianb Interesting, did flod mention using `innerHTML` to insert the tags? So far, we've always done

```el.textContent = i18n.getMessage('foo')```

but I don't really see how we can avoid using `innerHTML` for this string. As long as we're controlling the URL that gets inserted, I guess it should be fine from a security standpoint:

```
// raw strings in webextension.properties
privacyNotice = By using Firefox Screenshots, you agree to the Screenshots {linkTerms} and {linkPrivacy}.
linkTerms = Terms
linkPrivacy = Privacy Notice
```
```
// generated strings in addon/webextension/_locales/en_US/messages.json
  "privacyNotice": {
    "message": "By using Firefox Screenshots, you agree to the Screenshots $LINKTERMS$ and $LINKPRIVACY$.",
    "placeholders": {
      "linkterms": {
        "content": "$1"
      },
      "linkprivacy": {
        "content": "$2"
      }
    }
  },
```

```js
// js code:
const PRIVACY_URL = "https://whatever";
const TERMS_URL = "https://whatever"; // maybe these come from the manifest?
let privacyLink = "<a href='" + PRIVACY_URL + "'>" + i18n.getMessage("linkPrivacy") + "</a>";
let termsLink = "<a href='" + TERMS_URL + "'>" + i18n.getMessage("linkTerms") + "</a>";
let msg = i18n.getMessage("privacyNotice", termsLink, privacyLink);
privacyEl.innerHTML = msg;
```

**Comment by ianb** [comment]
We could use a sentinel to avoid innerHTML, like:

```js
let text = i18n.getMessage("termsAndPrivacyMessage", "!!!TERMS!!!", "!!!PRIVACY!!!");
let linkTexts = {
  TERMS: i18n.getMessage("linkTerms"),
  PRIVACY: i18n.getMessage("linkPrivacy")
};
let linkUrls = {
  TERMS: `${getBackend()}/terms`,
  PRIVACY: `${getBackend()}/privacy`
};
let parts = text.split("!!!");
let container = document.querySelector("terms-container");
for (let part of parts) {
  let el;
  if (part == "TERMS" || part == "PRIVACY") {
    el = document.createElement("a");
    el.target = "_blank";
    el.href = linkUrls[part];
    el.textContent = linkTexts[part];
  } else {
    el = document.createElement("span");
    el.textContent = part;
  }
  container.appendChild(part);
}
```

**Comment by ianb** [comment]
Fixed in #2612
